code/game/objects/items/cards_ids.dm

```diff
@@ -271,12 +271,18 @@
 		CAT_MEDSUP = MEDIC_TOTAL_BUY_POINTS,
 	)
 
+/obj/item/card/id/dogtag
```

Comment:
Should be its own type like the rest instead of redefining dogtag.

Reply:
FC's id uses this

Reply:
So do PFCs and MPs.

Reply:
They don't have access but made it its own type anyways.